test/todomvc.md

```diff
@@ -0,0 +1,38 @@
+---
+layout: test
+title:  TodoMVC
+---
+
+## Summary
+
+The purpose of this exercise is to demonstrate your approach to quality from multiple angles.
+
+There is no time limit for this test, but we expect most applicants to complete the requirements in roughly 3-4 hours.
+
+If you make any assumptions about requirements, or use any online resources to solve a problem, please make note of these in your documentation or comments.
+
+Your solution will be evaluated internally by your potential co-workers. You should expect a response from us within two business days.
+
+## Acceptance criteria
+
+1. Using an test automation framework of your choice, write automation tests for the [TodoMVC application](http://todomvc.com/) that cover the following scenarios:
+    - Add a new item.
+    - Remove an existing item.
+1. Come up with additional testing scenarios, writing automated tests for each.
+1. Ensure your submission contains a `README.md` file that includes:
+    - Instructions on how to run your automated test suite.
+    - Answers to the following questions:
+        - If you joined a team of six engineers as the only engineer with testing experience and could implement any process you would like, how would you ensure that the team delivers high quality software?
+        - If you were tasked with writing testing notes for this acceptance criteria, what would you write?
+1. Upload your submission to a public repository.
+1. Send the repository link to: `amp HYPHEN tech HYPHEN test HYPHEN submissions HYPHEN group AT fairfaxmedia DOT com DOT au`.
+
+## Developer notes
+
+1. You may choose to run your tests against a public [example TodoMVC application](http://todomvc.com/examples/react/), or spin up the TodoMVC application locally. In either case, your submission should be well documented and reproducible by engineers evaluating your test.
+1. Limit yourself when coming up with additional tests, don't spend an unreasonable amount of time here!
```

Comment:
I'd suggest waiting until this became a problem before putting this line in. Im quite interested in seeing what people think is the right level of testing by default, as well as what negative cases they thought to cover etc.
If anyone changes the code to add in preconditions / postconditions and invariants, and then runs property based tested against it, then let me know - I want to be part of the interview !!!

Reply:
Done!